rebar.config

```diff
@@ -7,7 +7,7 @@
 	{eradius, "2.2.4"},
 	{regine, "1.1.0"},
 	{cut, "1.0.3"},
-	{gtplib, "3.0.0"}
+	{gtplib, {git, "https://github.com/travelping/gtplib.git", {tag, "3.2.0"}}}
```

Comment:
can't you use the hex release instead?

Reply:
I didn't know a hex package was preferred to a tag in source